src/main/resources/subject-contact-details/translation.json

```diff
@@ -1,56 +1,56 @@
+      "emailAddressLabel": "Email address",
+      "emailAddressHint": "",
+      "emailAddress": "Insert email address",
+      "contactNumberLabel": "Contact number",
+      "contactNumberHint": "",
+      "agreeContactLabel": "I agree to be contacted about this case using the details I have provided"
     },
-    "errors":{
-        "en": {
-            "subjectEmailAddress": {
-                "required": "Please enter a valid email address",
-                "invalid": "Every email address consists of 3 elements: local-part, @ symbol (pronounced as “at”), and a domain name"
-            },
-            "subjectContactNumber": {
-                "required": "Please enter a valid contact number",
-                "invalid": "Please enter a valid contact number"
-            },
-            "subjectAgreeContact": {
-                "required" : "Please agree to be contacted using the details provided to continue"
-            }
-        },
-        "cy": {
-            "subjectEmailAddress": {
-                "required": "Nodwch gyfeiriad e-bost dilys",
-                "invalid": "Mae pob cyfeiriad e-bost yn cynnwys tair elfen: rhan lleol, symbol @ (a ynganir fel “at”), ac enw parth"
-            },
-            "subjectContactNumber": {
-                "required": "Nodwch rif cyswllt dilys",
-                "invalid": "Nodwch rif cyswllt dilys"
-            },
-            "subjectAgreeContact": {
-                "required" : "Dylech gytuno i’r llys gysylltu â chi gan ddefnyddio’r manylion a ddarparwyd cyn y gallwch barhau"
-            }
-        }
+    "cy": {
+      "continue": "Parhau",
+      "serviceName": "Apelio i’r Tribiwnlys Haen Gyntaf",
+      "title": "Rhowch fanylion cyswllt",
+      "line1": "Rhowch y manylion cyswllt sydd fwyaf addas ar gyfer yr achos hwn.",
+      "emailAddressLabel": "Cyfeiriad e-bost",
+      "emailAddressHint": "",
+      "emailAddress": "Nodwch gyfeiriad e-bost",
+      "contactNumberLabel": "Rhif cyswllt",
+      "contactNumberHint": "",
+      "agreeContactLabel": "Rwy’n cytuno i’r llys gysylltu â mi am yr achos hwn gan ddefnyddio’r manylion rwyf wedi’u darparu"
     }
+  },
+  "errors": {
+    "en": {
+      "subjectEmailAddress": {
+        "required": "Please enter an email address",
+        "invalid": "Please enter a valid email address. Every email address consists of 3 elements: local-part, @ symbol (pronounced as “at”), and a domain name"
+      },
+      "subjectContactNumber": {
+        "required": "Please enter a contact number",
+        "invalid": "Please enter a valid contact number"
+      },
+      "subjectAgreeContact": {
+        "required": "Please agree to be contacted using the details provided to continue"
+      }
+    },
+    "cy": {
+      "subjectEmailAddress": {
+        "required": "Rhowch gyfeiriad e-bost",
+        "invalid": "Nodwch gyfeiriad e-bost dilys. Mae pob cyfeiriad e-bost yn cynnwys tair elfen: rhan lleol, symbol @ (a ynganir fel “at”), ac enw parth"
```

Comment:
shouldn't "at" be translated to "yn" for welsh?

Reply:
yes, that's right. I think 'at' should be translated to welsh as well.

Reply:
https://tools.hmcts.net/jira/browse/DTSSTCI-36 This is the ticket addressing the translation of this particular portion of the text. It seems to suggest that "at" is the correct way to go in the attached files. However I could check with the BA?

Reply:
Yep, cross check with BA to double check.